valhalla/meili/routing.h

```diff
@@ -132,6 +132,7 @@ class Label : public sif::EdgeLabel {
   float target_;
 
   // Turn cost since origin (including the turn cost of this edge segment)
+  // TODO: can we rely on the built in transition_cost_ in the base class?
   float turn_cost_;
```

Comment:
sadly this turn_cost is cummulative so using the regular edge labels doesnt make sense at this time